pkg/graph/schema/types/user_view_customization.graphql

```diff
@@ -0,0 +1,35 @@
+enum ViewCustomizationType {
+  MY_MODEL_PLANS
+  ALL_MODEL_PLANS
+  FOLLOWED_MODELS
+  MODELS_WITH_CR_TDL
+  MODELS_BY_OPERATIONAL_SOLUTION
+}
+
+type UserViewCustomization {
+  id: UUID!
+  userAccount: UserAccount!
+  viewCustomization: [ViewCustomizationType!]!
+  possibleOperationalSolutions: [OperationalSolutionKey!]!
+  createdBy: UUID!
+  createdByUserAccount: UserAccount!
+  createdDts: Time!
+  modifiedBy: UUID
+  modifiedByUserAccount: UserAccount
+  modifiedDts: Time
+}
+
+input UserViewCustomizationChanges @goModel(model: "map[string]interface{}") {
+  viewCustomization: [ViewCustomizationType!]!

```

Comment:
I opted to just make these non-nullable to enforce both needing to be passed at all times. We could make them nullable, but I think since we're going to be following a `fetch then update` approach on the FE, making them non-nullable didn't seem like it'd hurt anything.

I can make them nullable and it shouldn't affect much, though, since we're using `applychanges`!

Reply:
Sounds good!